### PR TITLE
update Tests

### PR DESCRIPTION
**Pull request by RaggiTech**
# Changed log

- Tests methods has been updated.

@peter279k, please review this PR.

### CODE REVIEW

**Comment by peter279k** [comment]
As you can see, the test is failed. And Many failed assertions and exceptions happen.

**Comment by RaggiTech** [comment]
> As you can see, the test is failed. And Many failed assertions and exceptions happen.

ok i will work on it

File: tests/TranslatableTest.php [hunk 1]
```diff
@@ -8,227 +8,124 @@
...
-            'description' => 'post_test_description',
-            'keywords' => 'post_test_keyword',
-        ]);
-        $post = Post::create([
-            'id' => 1,
-            'published'=> true,
-        ]);
-
-        $actual = $post->translate('en')->title;
+        $this->post->translate('en')->title = 'post_test_title';
```
Comment (peter279k) [inline tests/TranslatableTest.php:58]: Do you forget to call the `$this->post->save()` method?

Comment (RaggiTech) [reply tests/TranslatableTest.php:58]: yes i think so

**Pull request closed without merging**